test_conformance/images/clCopyImage/test_copy_generic.cpp

```diff
@@ -632,5 +620,16 @@ int test_copy_image_generic( cl_context context, cl_command_queue queue, image_d
         return error;
     }
 
+    /**
+     * Call clFinish() to ensure the unmap call completes.
+     */
+    error = clFinish(queue);
+    if (error != CL_SUCCESS)
+    {
+        log_error("ERROR: clFinish() failed to return CL_SUCCESS: %s\n",
+                  IGetErrorString(error));
+        return -1;
```

Comment:
Any reason not to return the error here?

Reply:
ah yes, thank you,  it should return the error here. Uploaded a fix.